lib/collections/schemas/intlfield.js

```diff
@@ -0,0 +1,64 @@
+import { SimpleSchema } from "meteor/aldeed:simple-schema";
+
+/**
+* Reaction Schemas Intlfield
+*/
+
+const defaultAttributeOptions = {
+  type: String,
+  optional: true,
+  defaultValue: ""
+}
+
+const langs = [
+  'ar', 'bg', 'cs', 'de', 'el', 'en', 'es', 'fr', 'he', 'hr', 'hu', 'it', 'my',
+  'nb', 'nl', 'pl', 'pt', 'ro', 'ru', 'sl', 'sv', 'tr', 'vi', 'zh',
+]
+
+function createI18nObject ( string = "" ) {
+  const obj = {}
+
+  for ( let lang of langs ) {
+    obj[lang] = string
+  }
+
+  return obj
+}
+
+function autoValue () {
```

Comment:
needs jsdoc

Reply:
I'm gonna add JSDocs as soon as everything is working flawless. I can give this case as an example, I don't even know if we're gonna stick with this `autoValue` method.